### PR TITLE
Re-submit pull-request #9

### PR DESCRIPTION
**Pull request by mdaxtman**
This PR addresses all issues from Pull-request #9. It Adds radium grid, altered breakpoints and gutters removed unneeded React components. 
https://github.com/FormidableLabs/builder-docs/pull/9

@coopy @paulathevalley

### CODE REVIEW

**Comment by coopy** [comment]
Don't worry about the failing build for this PR – once it's merged, I will rebase this branch https://github.com/FormidableLabs/builder-docs/pull/12 off of master and fix any remaining lint issues.

Please do replace single quotes with doubles where you find them, first..

**Comment by mdaxtman** [comment]
@coopy single quotes gone

File: package.json [hunk 1]
```diff
@@ -24,9 +24,7 @@
   },
   "homepage": "https://github.com/FormidableLabs/builder-docs#readme",
   "devDependencies": {
-    "babel-core": "6.3.26",
-    "babel-loader": "6.2.1",
-    "builder": "https://github.com/FormidableLabs/builder",
+    "builder": "2.5.0",
```
Comment (coopy) [inline package.json]: Keep this pegged to `"builder": "FormidableLabs/builder",` so we get any updates the master README.

**Comment by coopy** [comment]
:+1:

**Pull request merged**